packages/cw1155/src/msg.rs

```diff
@@ -0,0 +1,43 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{Binary, HumanAddr, Uint128};
+
+pub type TokenId = String;
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum Cw1155HandleMsg {
+    /// TransferFrom is a base message to move tokens
```

Comment:
Interesting, there is no specific `Transfer` message, but if from is None, then we send from the `info.sender` account? What happens if the `to` field is `None`? I assume that is required, like `SendFrom.contract`

I'm a bit surprised to see Options here, can you just add a line of comment explaining what they do

Reply:
> Both from and to are Option, if from is None, it means mint new tokens, if to is None, it means burn tokens. They must not both be None at the same time.

In the spec there's a line to explain, `Option` is used to mean minting(from is `None`) and burning(to is `None`).